pyvecorg/data/meta.yml

```diff
@@ -45,6 +45,7 @@ reg_no:
     cs: IČO
     en: Reg. No.
   value: 22746668
+  link: https://or.justice.cz/ias/ui/vypis-sl-firma?subjektId=760829
```

Comment:
I'm using justice.cz here instead of ARES. ARES has an English version, but it is not useful compared to justice.cz as it contains like... three lines of information. I feel like linking justice.cz, even though it's all in Czech, makes more sense here. Also, most of the companies interacting with Pyvec are going to be of Czech origin anyway.